tests/test_processors.py

```diff
@@ -125,6 +126,95 @@ def test_repr_native_str(self, rns):
         assert 2 == cnt
 
 
+class TestLogfmtRenderer:
```

Comment:
Try to import logfmt above and then protect this whole class:

```suggestion
@pytest.mark.skipif(logfmt is None, reason="Needs logfmt package.")
class TestLogfmtRenderer:
```

Reply:
Fixed by 44a19e6 (drop `logfmt` dependency).